jovian/utils/clone.py

```diff
@@ -37,9 +38,17 @@ def get_gist(slug, version, fresh):
     else:
         url = _u('gist/' + slug + _v(version))
     res = get(url, headers=_h(fresh))
+    
     if res.status_code == 200:
         return res.json()['data']
-    raise Exception('Failed to retrieve Gist: ' + pretty(res))
+    elif res.status_code == 401:
+        # User need to login first
```

Comment:
Can remove this comment, as the log would convey the same.

Reply:
Removed.